htdocs/comm/propal/card.php

```diff
@@ -823,8 +823,15 @@
 				$type = $prod->type;
 				$fk_unit = $prod->fk_unit;
 			} else {
-				$pu_ht = price2num($price_ht, 'MU');
-				$pu_ttc = price2num(GETPOST('price_ttc'), 'MU');
+				if (!empty($conf->multicurrency->enabled)) {// we have to preserve all precision for the conversion as editline
```

Comment:
The pricenum(.., 'MU') means the user as setup dolibarr to be sure that into database we will not have more than x decimals saved (5 by default).
So why changing this ? Can you provide an example of use case that will be wrong ?